packages/build/src/core/normalize_flags.ts

```diff
@@ -40,7 +40,8 @@ export type ResolvedFlags = {
   context: 'production' | string
   statsdOpts: { host?: number; port: number }
   bugsnagKey?: string
-  tracing: TracingOptions
```

Comment:
We actually perform the transformation and apply the default values to `tracingOpts` 🤷 this makes sure we use that.